### PR TITLE
Add release notes for dns cache and dns app id plugins

### PR DESCRIPTION
**Pull request by agrawalkaushik**
DOCS-252 #time 20m

### CODE REVIEW

File: docs/plugin_dns_app_id.md [hunk 1]
```diff
@@ -341,6 +341,22 @@ exit
...
+
+- **PLUGIN-1452**  DNS cache services constantly fail on system startup
+
+  _**Resolution:**_ The DNS cache systemd services will be deferred until the 128T services are running and stable.
+
+- **PLUGIN-1461**  Config generation for the plugin failing in the Bonsai mode
+
+  _**Resolution:**_ Correctly handle the config generation for routers where the DNS cache plugin is not enabled during bonsai config generation
+
+- **PLUGIN-1494**  no route being injected into FIB table for dns-app-id custom apps.
```
Comment (BenMatase) [inline docs/plugin_dns_app_id.md:356]: Capital N here to start the sentence and remove the period at the end for consistency

File: docs/plugin_dns_app_id.md [hunk 2]
```diff
@@ -341,6 +341,22 @@ exit
...
+
+#### Issues Fixed
+
+- **PLUGIN-1452**  DNS cache services constantly fail on system startup
+
+  _**Resolution:**_ The DNS cache systemd services will be deferred until the 128T services are running and stable.
+
+- **PLUGIN-1461**  Config generation for the plugin failing in the Bonsai mode
+
+  _**Resolution:**_ Correctly handle the config generation for routers where the DNS cache plugin is not enabled during bonsai config generation
```
Comment (BenMatase) [inline docs/plugin_dns_app_id.md:354]: Capitalize Bonsai or do customers know/care about Bonsai?

**Review by BenMatase — CHANGES_REQUESTED** [review]

File: docs/plugin_dns_app_id.md (hunk 2 above)
Comment (MichaelBaj) [reply docs/plugin_dns_app_id.md:354]: Bonsai should be excluded from the release notes. Instead refer to the release in which Bonsai was introduced. e.g., >= 5.3

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/plugin_dns_app_id.md [hunk 3]
```diff
@@ -341,15 +341,31 @@ exit
...
+- **PLUGIN-1494**  No route being injected into FIB table for dns-app-id custom apps.
+
+  _**Resolution:**_ Automatically enable the module mode on router with dns-app-id config enabled.
+
 ### Release 3.1.3
 
 #### Issues Fixed
 
-- **PLUGIN-1461**  Config generation for the plugin failing
+- **PLUGIN-1461**  Config generation for the plugin failing.
```
Comment (BenMatase) [inline docs/plugin_dns_app_id.md:364]: We def aren't consistent with the periods at the end of sentences even further below. Dunno if you want to make it consistent or if this is sufficient

**Review by BenMatase — APPROVED** [review]

File: docs/plugin_dns_app_id.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/plugin_dns_app_id.md:364]: Honestly, if I chased the list/punctuation issue, I would be making changes to every release notes file all the way back to the beginning. As a writer I notice it (along with the oxford comma) and I know what the usage is supposed to be, but most folks don't pick up on it. You are one of the few that do, Ben. I try to keep the new stuff punctuated correctly.

**Comment by agrawalkaushik** [comment]
Thanks @Chr1st0ph3rTurn3r for handling the review comments - much appreciated!

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**